cgrok/grok.go

```diff
@@ -2,8 +2,7 @@ package grok
 
 /*
 #cgo CFLAGS: -I. -std=gnu99
-#cgo windows CFLAGS: -IC:/msys64/mingw64/include
```

Comment:
We're removing this line because we don't need the `portablexdr` headers that would normally be installed there?